### PR TITLE
Fix "missing-exports-condition" svelte warning

### PR DESCRIPTION
**Pull request by nielk**
Hi,

I use your package on my project and I get this warning message:

> 05:55:15 [vite-plugin-svelte] WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.
> 
> locally-unique-id-generator@0.1.5
> 
> Please see https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition for details.

This MR fixes this "issue" by adding a `"svelte"` field into the `"exports"` field in the package.json. I kept the existing `"svelte": "./dist/locally-unique-id-generator.esm.js",` for backward compatibility.

I've also updated project dependencies, I've checked that nothing has been broken.

I've also upgraded the "version" to "0.1.6".

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,12 +10,13 @@
...
       "require": "./dist/locally-unique-id-generator.js",
-      "import": "./dist/locally-unique-id-generator.esm.js"
+      "import": "./dist/locally-unique-id-generator.esm.js",
+      "svelte": "./dist/locally-unique-id-generator.esm.js"
     },
     "./package.json": "./package.json"
   },
   "scripts": {
-    "build": "rimraf dist && rollup -c rollup.config.js && tsc && mv src/*.d.ts dist && rm src/*.js*",
+    "build": "rimraf dist && rollup -c rollup.config.js && tsc && mv src/*.d.ts dist && rimraf -g src/*.js* && rimraf dist/src",
```
Comment (nielk) [inline package.json:19]: A `.d.ts` was moved to `dist/src/xxx.d.ts`, probably a side-effect of upgraded packages… So now I remove it after each build.

File: package.json [hunk 2]
```diff
@@ -32,10 +33,11 @@
   },
   "homepage": "https://github.com/rozek/locally-unique-id-generator#readme",
   "devDependencies": {
+    "@rollup/plugin-terser": "^0.4.4",
     "@rollup/plugin-typescript": "^8.2.1",
-    "rimraf": "^3.0.2",
+    "rimraf": "^6.0.1",
     "rollup": "^2.52.3",
-    "rollup-plugin-terser": "^7.0.2",
```
Comment (nielk) [inline package.json:38]: Deprecated package, instead it is recommended to use `"@rollup/plugin-terser"`

File: package.json [hunk 3]
```diff
@@ -32,10 +33,11 @@
...
-    "typescript": "^4.3.4"
+    "tslib": "^2.7.0",
```
Comment (nielk) [inline package.json:40]: Needed to run `@rollup/plugin-terser`

**Review by rozek — APPROVED** [review]
Thanks for your contribution!

**Comment by rozek** [comment]
I've used your issue as a trigger to modernize how I build my numerous modules. This "library" has served as a starter - the version has changed (and it has been published again) but you should not observe any relevant changes. Please contact me in the case of any problems.

**Pull request merged**